modules/repository-gcs/src/test/java/org/elasticsearch/repositories/gcs/GoogleCloudStorageBlobContainerRetriesTests.java

```diff
@@ -570,6 +581,54 @@ public void testCompareAndExchangeWhenThrottled() throws IOException {
         container.delete(randomPurpose());
     }
 
+    public void testContentsChangeWhileStreaming() throws IOException {
+        GoogleCloudStorageHttpHandler handler = new GoogleCloudStorageHttpHandler("bucket");
+        httpServer.createContext("/", handler);
+        final int enoughBytesToTriggerChunkedDownload = Math.toIntExact(ByteSizeValue.ofMb(30).getBytes());
```

Comment:
Do you mind adding a comment to explain briefly how the size of 30mb is decided?

Reply:
Done in ce03daeab707678bfa7406a3c2af571d74fa8112